Apps/CocoaPods-FCM/src/View/DashboardView.swift

```diff
@@ -49,7 +49,7 @@ struct DashboardView: View {
                     }
                     Text("What would you like to test?")
                     Group {
-                        InAppMessageViewRepresentable(elementId: "dashboard-announcement", onActionClick: { _, _, _ in
+                        InlineInAppMessage(elementId: "dashboard-announcement", onActionClick: { _, _, _ in
```

Comment:
I am going to ask the team for feedback on what the name of the SwiftUI inline View could be. As Aman has mentioned before, there is a high chance of clashing with the UIKit names being used. Going to have to find a unique name just for SwiftUI.

Reply:
Based on the slack conversation, I suggest the following names :
`InlineMessageView` for UIKit 
`InlineMessageViewRepresentable` or `InlineMessageViewComponent` for SwiftUI. I have given reasoning on our slack conversation in case it helps.

Reply:
I'll rename the class in a follow-up PR if that's OK with you.

Reply:
That works for me. I will go ahead and approve this PR to unblock you.